### PR TITLE
Introduce simple filter with button and text input <?.x.?>

### PR DESCRIPTION
**Pull request by karelhala**
### New filter component
This PR introduces new component called `SimpleTableFilter` for filtering in tables
 * input -> onChange has callback `onFilterChange` to filter without need of pressing button (not good for server generated reports)
 * button -> onClick has callback `onButtonClick` this callback will get active filter

#### Props
```javascript
{
  buttonTitle: PropTypes.string,
  placeholder: PropTypes.string,
  className: PropTypes.string,
  onButtonClick: PropTypes.func,
  onFilterChange: PropTypes.func
}
```

#### Example
screenshot from 2018-07-17 15-27-12

### CODE REVIEW

File: src/PresentationalComponents/SimpleTableFilter/SimpleTableFilter.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  }
+}
+
+SimpleFilter.propTypes = {
+  buttonTitle: PropTypes.string,
+  placeholder: PropTypes.string,
+  className: PropTypes.string,
+  onButtonClick: PropTypes.func,
+  onFilterChange: PropTypes.func
+}
```
Comment (ryelo) [inline src/PresentationalComponents/SimpleTableFilter/SimpleTableFilter.js]: Do we need to assign default values to any of these propTypes or require anything?

Comment (ryelo) [inline src/PresentationalComponents/SimpleTableFilter/SimpleTableFilter.js]: Do we want to require any of these? Couldn't someone just clear out the placeholder string or buttonTitle and have blank spaces?

Comment (karelhala) [reply src/PresentationalComponents/SimpleTableFilter/SimpleTableFilter.js]: These proptypes are for users of this component so they know what props and what values to pass into. Of course if somebody doesn't want to have placeholder he can pass empty string, if somebody doesn't want to have button they can just pass ampty string into `buttonTitle` and no button will be visible.

**Pull request merged**